src/types/api.types.ts

```diff
@@ -48,7 +48,7 @@ export const sessionSchema = z.object({
   meta_data: metaDataSchema,
   name: z.string(),
   owner_id: z.string(),
-  platform: z.string(),
+  platform: z.string().nullable(),
```

Comment:
i see the problem with having platform as null now... maybe we should just give an option "No Platform" ? Not sure